source/_config.yml

```diff
@@ -10,8 +10,7 @@ blogurl: https://blog.patternfly.org
 exclude: ["components/patternfly/tests-src"]
 navbar-pf-navbar-brand-icon: true
 navbar-pf-alt-navbar-brand-icon: true
-navbar-logo-icon: '/assets/img/logo.svg'
-navbar-logo-alt-icon: '/assets/img/logo-alt.svg'
+navbar-logo-icon: '/assets/img/logo-alt.svg'
```

Comment:
navbar-logo-icon should point to /assets/img/logo.svg
navbar-logo-alt-icon should point to /assets/img/logo-alt.svg